src/Header.cpp

```diff
@@ -0,0 +1,37 @@
+#include "lift/Header.hpp"
+
+#include <string>
+
+namespace lift {
+
+Header::Header(
+    std::string_view name,
+    std::string_view value)
+{
+    m_header.reserve(name.length() + value.length() + 2);
+    m_header.append(name);
+    m_header.append(": ");
+    m_header.append(value);
+
+    auto colon_pos = m_header.find(':');
```

Comment:
Won't colon_pos always be name.length() or are we looking for colon in case someone added it to the header already??

Reply:
yeah, or at least we could cache m_header.size() before the append of ": "

Reply:
good point, i don't need to find on this, will change